package/MDAnalysis/coordinates/PDB.py

```diff
@@ -1051,6 +1051,8 @@ def _write_timestep(self, ts, multiframe=False):
            Writing now only uses the contents of the elements attribute
            instead of guessing by default. If the elements are missing,
            empty records are written out (Issue #2423).
+           Atoms are now checked for a chainID instead of being overwritten
```

Comment:
Could you be more specific? e.g. What happens if there is no chainID?

Reply:
@lilyminium I just realized i've been marking these conversations as resolved as I've addressed the change. Is that bad practice? Should I be letting you mark these as resolved or should I resolve them as they are addressed? Thanks.

Reply:
What you're doing now is good :-)

Reply:
Thanks @lilyminium ! :smile:  Is there anything else I need to do at this point, or are we just waiting on confirmation of if they want to do anything more than just use the chainID (if the attribute exists and is all alphanumeric) else last char of segid?